modules/package.json

```diff
@@ -56,22 +56,22 @@
         "clean": "git clean -f -d *"
     },
     "devDependencies": {
-        "@babel/core": "^7.21.0",
-        "@babel/preset-env": "^7.20.2",
-        "@babel/register": "^7.21.0",
-        "@babel/runtime": "^7.21.0",
+        "@babel/core": "^7.25.2",
+        "@babel/preset-env": "^7.25.3",
+        "@babel/register": "^7.24.6",
+        "@babel/runtime": "^7.25.0",
         "@open-wc/webpack-import-meta-loader": "^0.4.7",
-        "babel-loader": "^9.1.2",
+        "babel-loader": "^9.1.3",
         "babel-plugin-add-module-exports": "^1.0.4",
-        "babel-plugin-transform-import-meta": "^2.2.0",
-        "ilib-assemble": "^1.2.0",
+        "babel-plugin-transform-import-meta": "^2.2.1",
+        "ilib-assemble": "^1.2.1",
```

Comment:
This should be v1.3.0 to go with your new version!

Reply:
I updated it